src/PoissonSolvers/FFTOpenPoissonSolver.hpp

```diff
@@ -1141,6 +1203,124 @@ namespace ippl {
             }
             IpplTimings::stopTimer(ifftshift);
 
+        } else if (alg == Algorithm::DCT_VICO) {
+            Vector_t l(hr_m * nr_m);
+            Vector_t hs_m;
+            double L_sum(0.0);
+
+            // compute length of the physical domain
+            // compute Fourier domain spacing
+            for (unsigned int i = 0; i < Dim; ++i) {
+                hs_m[i] = pi * 0.5 / l[i];
```

Comment:
I may have missed it, but I assume `pi` here is `Kokkos::numbers::pi` (or similar).

Reply:
Is this resolved?